SPECS/cmake/cmake.spec

```diff
@@ -12,14 +12,10 @@ Source0:        https://github.com/Kitware/CMake/releases/download/v%{version}/%
 Source1:        macros.cmake
 Patch0:         disableUnstableUT.patch
 # We could use --system-curl instead of patching, but unfortuately curl isn't currently available in time during the toolchain build.
```

Comment:
I think we should remove stale comments.
Also is this comment valid for us for 3.0 ?

Reply:
Nope, these patches don't apply anymore, hence the comments aren't valid at the moment